games/t.yaml

```
@@ -604,7 +601,7 @@
   updated: 2017-10-23
   url: https://github.com/xesf/twin-e
 
-- name: Typescript Xenon
+- name: Xenon
```

```suggestion
- name: TypeScript Xenon
```
The name comes from the URL; please do not change